docs/concepts/identity.md

```diff
@@ -14,6 +14,7 @@ Here are some important aspects of identity contracts:
 
 - Identity contracts define how proofs of identity are verified and validated.
 - Applications decide which identities to accept. A contract can enforce specific identity types (e.g., Google accounts only) or support multiple sources simultaneously.
+- Identity providers must provide a verifiable signature to be secure.
```

Comment:
un peu trop simple ça je trouve.

A safe Identity provider should sign the whole blob transaction to ensure that the blobs in the transactions are indeed the ones the user wanted to do. One way of doing it is to let the user signs the blobs he agrees to execute, the identity contract should check that the signed blobs matches the transaction's blobs.